pkg/crc/network/proxy.go

```diff
@@ -133,6 +133,9 @@ func ValidateProxyURL(proxyUrl string) error {
 		return nil
 	}
 
+	if !strings.HasPrefix(proxyUrl, "https://") && !strings.HasPrefix(proxyUrl, "http://") {
+		return fmt.Errorf("Proxy URL '%s' is not valid: url should start with http:// or https://", proxyUrl)
```

Comment:
As per doc https://docs.openshift.com/container-platform/4.4/networking/enable-cluster-wide-proxy.html#enable-cluster-wide-proxy, under section `3. Configure the necessary fields for the proxy:` it is documented that `A proxy URL to use for creating HTTPS connections outside the cluster. If this is not specified, then httpProxy is used for both HTTP and HTTPS connections. The URL scheme must be http; https is currently not supported.` so we might need to only support `http` part

Reply:
Nice ! I changed the condition and added an explicit error message for https.